python/lsst/ap/association/assoc_db_sqlite.py

```diff
@@ -136,7 +136,7 @@ def source_record_to_value_list(self, source_record):
 
         Parameters
         ----------
-        source_record : afw.table.SourceRecord
+        source_record : lsst.afw.table.SourceRecord
```

Comment:
Not related to the Coord -> SpherePoint conversion.

Reply:
True. Are you requesting that it be on a separate commit?

Reply:
I'm fine with it, but do we have a policy on this?  I've been asked by other reviewers to do cleanups in separate commits, but I don't know if this is standard practice.

Reply:
it is standard practice.

Reply:
I'll just drop that change. I have a version of this file whose docs are cleaned up, but I don't want to mess up the merge of DM-13478 so I'll just hold on to it privately for now.